src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMailboxes.scala

```diff
@@ -60,10 +62,23 @@ object JmapMailboxes {
   def getMailboxesChecks(expectedNumberOfMailboxes: Int): Seq[HttpCheck] =
     getMailboxesChecks :+ assertNumberOfMailboxes(expectedNumberOfMailboxes)
 
+  def checkMailboxIdsHasNotChange: Seq[HttpCheck] =
```

Comment:
We should indicate it is system mailboxes, no?

Like `checkSystemMailboxIdsHaveNotChange`